bang/templates/items/costumeItem.html

```diff
@@ -0,0 +1,19 @@
+{% load tools %}
+{% load i18n %}
+
+{% with costume=item %}
+<a href="{{ costume.viewer_url }}" class="costume_link" data-ajax-url="{{ costume.ajax_viewer_url }}" data-ajax-title="{{ costume }}">
+    <div class="cos_image">
+        {% if costume.resolved_preview_image %}
+        <img alt="{{ costume }}" src="{{ costume.resolved_preview_image }}" height="200"/>
```

Comment:
Add `class="img-responsive"` which will prevent it from overflowing if the size is bigger than 100%.